src/openrct2/ride/Vehicle.cpp

```diff
@@ -3574,7 +3574,7 @@ static void vehicle_update_collision_setup(rct_vehicle* vehicle)
 }
 
 /** rct2: 0x009A3AC4, 0x009A3AC6 */
-static constexpr const LocationXY16 stru_9A3AC4[] = {
+static constexpr const CoordsXY stru_9A3AC4[] = {
```

Comment:
These are not coords. I'm not really sure what they represent yet but they aren't coords

Reply:
It looks like they are a collection of constant used to calculate coords for the vehicle crash

Reply:
The code using them is very hard to read, but it does look like they are indeed base values for vehicle crash coords.

Not sure if we should save them in a CoordsXY value - perhaps it's better to revert this change, for now at least.

Reply:
Ok, since it was the only change in that commit, I dropped it altogether.